config/samples/custom-rb.yaml

```diff
@@ -0,0 +1,26 @@
+# (c) Copyright [2021-2022] Micro Focus or one of its affiliates.
```

Comment:
General comment about these new files in config/samples.  Can we have them generated?  It'll be error prone to ensure they are kept in sync with the roles/rolebindings in the helm chart.  We can give them the default name (i.e. verticadb-operator-manager-role).  When using them in the new e2e script, we can use kustomize to change the name of the roles.